src/components/Inputs/Select.tsx

```diff
@@ -160,11 +160,12 @@ function SelectOptionsPopup<T extends OptionValue>({
   onBackgroundClick: () => void
   title?: string
 }) {
-  const handleEvent = (el: HTMLSelectElement) =>
+  const handleEvent = (el: HTMLSelectElement) => {
     handleOptionSelect({
       label: options[el.selectedIndex]?.label,
       value: el.value as T
     })
+  }
```

Comment:
I don't see why this change was necessary, but ok :)

Reply:
I remember seeing this during rebase; yeah it isn't